lib/helper/Protractor.js

```diff
@@ -874,6 +874,18 @@ class Protractor extends Helper {
     return client.switchTo().window(newHandle);
   }
 
+  /**
+   * Grab number of open tabs
+   *
+   * ```js
+   * I.grabNumberOfOpenTabs();
```

Comment:
This function is used across different helpers so I'd recommend to move the documentation to external file and include it. 

Create `grabNumberOfOpenTabs.md` inside https://github.com/Codeception/CodeceptJS/tree/master/docs/webapi

Place documentation there. Include this files here as 

```
{{> ../webapi/grabNumberOfOpenTabs }}
```
Also for grabbers, please provide an example with the variable actually assigned:

```js
const tabsNum = await I.grabNumberOfOpenTabs();
```

Reply:
Sure makes sense. Though I assumed you meant a `.mustache` file instead of `.md`.